docs/code-samples/instruments-config/0.0.3.conf

```diff
@@ -0,0 +1,32 @@
+https://github.com/digital-asset/daml-finance/releases/download/ContingentClaims.Core/1.0.0.99.20230615.1/contingent-claims-core-1.0.0.99.20230615.1.dar .lib/contingent-claims-core.dar
```

Comment:
I would re-use the same config file as before (although it's called `getting-started-config`. @brianweir-da thoughts on this?

Reply:
I agree, I think one config file seems better. Maybe renaming that config folder to `tutorials-config` and updating the `get-dependencies*` files in both `getting-started` and the new `instruments` folder?

Reply:
renaming is unfortunately not an option, as it would break the existing `getting-started` tutorial. 

One option is to
- rename the config as you suggest
- symlink the new file to the old location (for backwards-compatibility)

Reply:
> renaming is unfortunately not an option, as it would break the existing `getting-started` tutorial.

@brianweir-da Is that true? Or is the name of the folder only relevant for the "production commit" in the daml repo?

Reply:
> I would suggest
> 
> * leave `0.0.1` and `0.0.2` where they are for backwards-compatibility
> * create a new folder `tutorials-config` with the `0.0.3` config file to be used by both scripts
> * update `get-dependencies` to use `0.0.3` config

I did this now. If it looks fine, I will go ahead and merge.

Reply:
> > renaming is unfortunately not an option, as it would break the existing `getting-started` tutorial.

> @brianweir-da Is that true? Or is the name of the folder only relevant for the "production commit" in the daml repo?

Yes, that is correct. The folder name is part of the path that the `curl` command uses to download these files, meaning if you change that on `main` then all previous and current `get-dependencies.sh|.bat` scripts will fail.

Reply:
Thanks for the confirmation! Currently, the quickstarter in production is on v0.0.2, so when this PR is merged, it should not break anything because the old config files are still in the old folder, only the new v0.0.3 is in the new one, correct?

Reply:
Correct